content/en/blog/_posts/2023-08-29-Gateway-API-v080.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: blog
+title: "Gateway API v0.8.0: Introducing Service Mesh Support"
+date: 2023-08-29
+slug: gateway-api-v0-8
+---
+
+***Authors:*** Flynn (Buoyant), John Howard (Google), Keith Mattix (Microsoft), Michael Beaumont (Kong), Mike Morris (Independent), Rob Scott (Google)
+
+We are thrilled to announce the v0.8.0 release of Gateway API! With this
+release, Gateway API support for service mesh has reached [Experimental
+status][status], we've introduced CEL validation and a `Mesh` conformance
+profile, and we've made some changes to the GEP process. We look forward to
+your feedback!
+
+We're especially delighted to announce that Kuma 2.3+, Linkerd 2.14+, and
+Istio 1.16+ are all fully-conformant implementations of the Gateway API
+service mesh support.
+
+## Why add service mesh support to Gateway API?
```

Comment:
```suggestion
## Service mesh support in the Gateway API
```

Reply:
I think this should be covered now that I've copied everything from your PR back over here, let me know if I missed anything.

Reply:
I do like the more direct section heading suggestion better, think that's the last update!

Reply:
Looks like my last push missed some things, updated, thanks!